### PR TITLE
Feature/phenotype widgets

### PR DESCRIPTION
**Pull request by adamjohnwright**
I have created the phenotype page widgets:

- Associated anatomy
- GO term
- RNAi
- Transgene
- Variation

I have tested these URLs by looking at the resulting JSON. They seem to be fairly simple.

After this pull request, I will make small changes based on what I see on the staging website. After that I will request curators to take a look.

### CODE REVIEW

File: src/rest_api/classes/phenotype/widgets/associated_anatomy.clj [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+                    :assay
+                    (when-let [ahs (:anatomy-function/assay af)]
+                      (for [ah ahs]
+                        {:text (:ao-code/id (:anatomy-function.assay/ao-code ah))
+                         :evidence (obj/get-evidence ah)}))
+
+                    :phenotype
+                    (when-let [phs (:anatomy-function.phenotype/phenotype af)]
+                      (keys phs)
```
Comment (sibyl229) [inline src/rest_api/classes/phenotype/widgets/associated_anatomy.clj]: Maybe the keys need to be humanized to get rid of the namespace?

File: src/rest_api/classes/phenotype/widgets/variation.clj [hunk 2]
```diff
@@ -1,8 +1,57 @@
 (ns rest-api.classes.phenotype.widgets.variation
   (:require
-   [rest-api.classes.generic-fields :as generic]))
+    [rest-api.formatters.object :as obj :refer  [pack-obj]]
+    [rest-api.classes.generic-fields :as generic]))
+
+(defn variation-info [v obs]
```
Comment (sibyl229) [inline src/rest_api/classes/phenotype/widgets/variation.clj]: the `v` parameter might have been `phenotype`

File: src/rest_api/classes/phenotype/widgets/transgene.clj [hunk 3]
```diff
@@ -1,8 +1,43 @@
 (ns rest-api.classes.phenotype.widgets.transgene
   (:require
+    [rest-api.formatters.object :as obj :refer  [pack-obj]]
     [rest-api.classes.generic-fields :as generic]))
 
+(defn transgene-info [t obs]
```
Comment (sibyl229) [inline src/rest_api/classes/phenotype/widgets/transgene.clj]: `t` parameter might have been phenotype

File: src/rest_api/formatters/object.clj [hunk 4]
```diff
@@ -333,43 +333,105 @@
...
+
+   :genotype
+   (when-let [c (:anatomy-function.assay/condition holder)]
+     (first (:condition/genotype c)))
+
+   :Autonomous
+   (when-let [ss (:anatomy-function-info/autonomous holder)]
+     (for [s ss]
+       {:taxonomy "all"
+        :class "txt"
```
Comment (sibyl229) [inline src/rest_api/formatters/object.clj:382]: Might need to double check if the class should 'txt' or 'text'

Comment (adamjohnwright) [reply src/rest_api/formatters/object.clj:382]: This is how the object are coming from Catalyst API. I am leaving it the same for now. In the future we could look into making this more consistent.

**Pull request merged**